src/Lachain.Consensus/IPublicConsensusKeySet.cs

```diff
@@ -13,5 +13,6 @@ public interface IPublicConsensusKeySet
         PublicKeySet ThresholdSignaturePublicKeySet { get; }
         IList<ECDSAPublicKey> EcdsaPublicKeySet { get; }
         public int GetValidatorIndex(ECDSAPublicKey publicKey);
+        public byte[] ToBytes();
```

Comment:
Should we add corresponding `FromBytes` method too?

Reply:
I don't think so, because FromBytes is static method.